rails_util.gemspec

```diff
@@ -15,6 +15,6 @@ Gem::Specification.new do |s|
   s.files       = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
 
   s.add_development_dependency 'rspec'
-  s.add_dependency 'activesupport', '~> 5.0.0'
+  s.add_dependency 'activesupport', '~> 5.2'
```

Comment:
should I make this '~> 5' ? figure most projects are rails 5 or above?

Reply:
I think we'd want to preserve backwards compatibility as best we can. Seems like `~> 5` is a safer way to go.

Reply:
agreed! made that change

Reply:
Should the version range be omitted in that case? I think `~> 5` would equivalent to any version.

Reply:
I believe there's a version 6 out there which would be for rails 6.